frontend/src/locales/en.json

```diff
@@ -696,6 +696,67 @@
     "lastDefeated": "You were last defeated by: ",
     "tapAnywhere": "Tap anywhere to close",
     "kicked": "You were kicked from the arena!",
+    "atLeastOneChar": "You need at least one character to enter PvP!",
+    "arenaSignUpCaps": "ARENA SIGNUP",
+    "enterAndWin": "Enter the arena and win rewards ",
+    "equipSwordAndShield": "Equip a Sword and a Shield (optional).",
+    "weapons": "Weapons",
+    "noWeapons": "You have no weapons.",
+    "clear": "Clear",
+    "shields": "Shields",
+    "noShields": "You have no shields.",
+    "enterTheArena": "Enter the Arena",
+    "enterArenaWillCost": " Entering the Arena will cost you {formattedEntryWager} $SKILL.",
+    "playersCanAttackYou": " Players can attack you while you are in the Arena.",
+    "leavingWillCost": " Leaving the Arena will cost you {leavingArenaCost} $SKILL.",
+    "iUnderstand": "I understand.",
+    "enterArenaCaps": "ENTER ARENA",
+    "equipment": "Equipment",
+    "duelHistory": "Duel history",
+    "date": "Date",
+    "result": "Result",
+    "win": "Win",
+    "lose": "Lose",
+    "arena": "Arena",
+    "arenaTier": "Arena tier:",
+    "wagerLeft": "Wager left:",
+    "level": "Level",
+    "rank": "Rank",
+    "youHave": "You have ",
+    "toAcceptDuel": " to accept the duel",
+    "duelHasExpired": "Duel has expired.",
+    "inProgressCaps": "IN PROGRESS",
```

Comment:
As mentioned above, we shouldn't have all-caps values in translations, change it to "In progress" and use css to make it uppercase